satellitevu/apis/otm.py

```diff
@@ -243,6 +243,86 @@ def get_price(
         )
         return response.json()
 
+    def post_search(
```

Comment:
Is "post" here because you're using the HTTP request type? That isn't a pattern followed by the other endpoints - there's a few "Gets", but also some "Lists" and "Creates". "post_search" doesn't immediately tell me what a post search is!

Reply:
Primarily yes, in the [docs](https://api.qa.satellitevu.com/otm/v2/docs#operation/post_search__contract_id__search__post) it is called `Post Search` which is why I named it like this. Also it kind of mimics the convention with stac-fastapi (but that is probably just a happy coincidence more than anything). 

Happy to change to just `search` if you feel it's misleading?

Reply:
I guess the title and wording feels odd compared to the other examples there. "List all tasking orders",  "List all feasibility requests", but "Post Search" without saying what the search is for :)

Reply:
agreed it is very vague- it's a search across all feasibility and orders (requests and responses) but tbh i'm not what a concise way of phrasing that is

Reply:
i'm unsure about `client.otm_v2.otm_search()` as opposed to `client.otm_v2.search()`